### ISSUE TITLE
IE compatibility mode is not avaiable

### ISSUE THREAD

**Issue by ksjun** [testem/testem#995]
'testem launchers' doesn't list up IE7-9 as README.md. Only 'IE' is displayed.

And 'testem ci -l ie7' shows 'Launcher ie9 not found'.

**Comment by johanneswuerbach** [comment]
Yes, this has been previously supported, but has been removed at some point due to difficulties implementing it as compatibility mode requires the browser to receive a specific header as no command line flag is available to toggle it.

Help bringing it back is very welcome.
